### ISSUE TITLE
Wrong timezone in tray

### ISSUE THREAD

**Issue by babysas** [White-Tiger/T-Clock#64]
![tz](https://cloud.githubusercontent.com/assets/22335466/18695622/b7e68bbc-7fbb-11e6-82a6-715dadcea547.jpg)

Time Zone when added to clock showing wrong. 
Screenshot is attached.

**Comment by White-Tiger** [comment]
yep... the doc also says: `Note: doesn't work currently, fixes or suggestions on how to name timezones are welcome.`

"PDT / PST" is simply the default value from Windows... all that T-Clock does is to call _get_tzname()  https://msdn.microsoft.com/en-us/library/4ssfs1ya.aspx

**Comment by bilogic** [comment]
```_get_tzname()``` does not get the system's timezone, use ```GetTimeZoneInformation``` instead. See #155.
